capstone-frontend/src/components/ListPage.jsx

```diff
@@ -77,33 +81,52 @@ class ListPage extends Component {
       :
       (this.state.empty
         ?
-        (<h1 className="text-center mt-4">Booklist has No Books</h1>)
+        (
+          <div>
+            <h2 className="mt-3">{this.state.bookList.name}</h2>
+            <hr className="light-gray-border" />
+            <h3 className="text-center mt-4">Booklist has No Books</h3>
+          </div>
+        )
         : (
-          <div className="text-center mt-4">
-            {
-              this.state.gBooks.map(gBook =>
-                <div key={gBook.id + this.props.match.params.id}>
-                  <a className="text-decoration-none text-body" href={gBook.canonicalVolumeLink}>
-                    <div>
-                      <img src={gBook.thumbnailLink} alt={gBook.title} />
+          <div>
+            <h2 className="mt-3">{this.state.bookList.name}</h2>
+            <hr className="light-gray-border" />
+
+            <div>
+              {
+                this.state.gBooks.map(gBook =>
+                  <Row className="text-center border m-5 bg-light light-gray-border" key={gBook.id + this.props.match.params.id} >
```

Comment:
Is using the `gBook.id` by itself as the key not sufficient? It should be unique right?

Reply:
Yes, but multiple BookLists can have the same book should causing React to throw an error of non unique key.